lib/interactor/builder.ex

```diff
@@ -0,0 +1,130 @@
+defmodule Interactor.Builder do
+
+  @moduledoc """
+
+
+  The Interactor.Builer module functionality and code is **heavily** influenced
+  and copied from the Plug.Builder code.
+  TODO.
+
+  Example:
+
+      def Example.CreatePost do
+        use Interactor.Interaction
+        import Ecto.Changeset
+
+        interactor :post_changeset
```

Comment:
I understand the analogy, but I think maybe what I'm hung up on is "what is an interactor?". Is `CreatePost` the interactor or is `post_changeset`? My understanding was that `CreatePost` is the interactor, and this DSL is telling the interactor to perform a task. I'm basing that on https://github.com/collectiveidea/interactor#what-is-an-interactor.